### DISCUSSION TITLE
Bed Visualizer GCODE

### DISCUSSION THREAD

**Discussion by NotMe2210** [Jyers/Marlin#244] [Q&A] [answered]
I'm using the manual 3x3 mesh leveling and wanted to use the Visualizer plugin to show my bed. Unfortunately I'm not very firm with GCODE and don't know what command to put where to get results shown. I read about putting the M420 S1 command in the start code but what is meant by start code exactly? What do I have to type in the Bed Visualizers Settings GCODE for updating the mesh?

**Comment by clintiepoo** [comment]
Check out these samples to her bed visualizer “looking” every time you level. 
https://github.com/jneilliii/OctoPrint-BedLevelVisualizer/blob/76c488744e36f4cd05218ceb22257068e2aabab0/wiki/gcode-examples.md#bilinear-bed-levelling

I don’t remember the exact commands, but I’m doing a “mesh activate” and “3 point tilt” before each print. Check Marlin’s ubl page for the different commands.

**Comment by Jyers** [comment, marked as answer]
This script can be used in the plugin to retrieve the current mesh:

@BEDLEVELVISUALIZER
M420 V1
